examples/tutorials/earth-relief.py

```diff
@@ -0,0 +1,110 @@
+# figure with a Winkel Tripel projection. For a list of available projections,
+# see :gmt-docs:`cookbook/map-projections.html`.
+
+fig = pygmt.Figure()
+fig.grdimage(grid=grid, projection="R12c")
+fig.show()
+
+########################################################################################
+# Set a color map
+# ---------------
+#
+# :meth:`pygmt.Figure.grdimage` takes the ``cmap`` argument to set the CPT of the
+# figure. Examples of common CPTs for Earth relief are shown below.
+# A full list of CPTs can be found at :gmt-docs:`cookbook/cpts.html`.
+
+########################################################################################
+#
+# *geo*
+
+fig = pygmt.Figure()
+fig.grdimage(grid=grid, projection="R12c", cmap="geo")
+fig.show()
+
+########################################################################################
+#
+# *relief*
+
+fig = pygmt.Figure()
+fig.grdimage(grid=grid, projection="R12c", cmap="relief")
+fig.show()
+
+########################################################################################
+#
+# *etopo1*
+
+fig = pygmt.Figure()
+fig.grdimage(grid=grid, projection="R12c", cmap="etopo1")
+fig.show()
+
+########################################################################################
+# Add a color bar
+# ---------------
+#
+# The :meth:`pygmt.Figure.colorbar` method displays the CPT and the associated Z-values
+# of the figure, and by default uses the same CPT set by the ``cmap`` argument
+# for :meth:`pygmt.Figure.grdimage`. The ``frame`` argument for
+# :meth:`pygmt.Figure.colorbar` can be used to set the axis intervals and labels. A
+# list is used to pass multiple arguments to ``frame``. In the example below,
+# ``"a2500"`` sets the axis interval to 2,500, ``"x+lElevation"`` sets  the x-axis
+# label, and ``"y+lm"`` sets the y-axis label.
+
+fig = pygmt.Figure()
+fig.grdimage(grid=grid, projection="R12c", cmap="geo")
+fig.colorbar(frame=["a2500", "x+lElevation", "y+lm"])
+fig.show()
+
+########################################################################################
+# Create a region map
+# -------------------
+#
+# In addition to providing global data, the ``region`` argument for
+# :meth:`pygmt.datasets.load_earth_relief` can be used to provide data for a specific
+# area. The ``region`` argument is required for resolutions at 5 minutes or lower, and
+# accepts a list (as in the example below) or a string. The geographic ranges are
+# passed as *x-min*/*x-max*/*y-min*/*y-max*.
+#
+# The example below uses data with a 5 minute resolution, and plots it on a
+# 15 centimeter figure with a Mercator projection and a CPT set to *geo*.
+# :meth:`pygmt.Figure.basemap` is used to add a frame to the figure, using the argument
+# ``"a"`` for the ``frame`` parameter.
+
+grid = pygmt.datasets.load_earth_relief(resolution="05m", region=[-14, 30, 35, 60])
+fig = pygmt.Figure()
+fig.grdimage(grid=grid, projection="M15c", cmap="geo")
+fig.basemap(frame="a")
```

Comment:
We can merge these two commonds into one.
```suggestion
fig.grdimage(grid=grid, projection="M15c", frame="a", cmap="geo")
```